src/05-derivatives/02-common-data-types.md

```diff
@@ -71,9 +71,9 @@ The `space` entity may take any value in [Image-Based Coordinate Systems][coords
 If the `space` entity is omitted, or the space is not in the [Standard template
 identifiers][templates] table, then the `SpatialReference` metadata is REQUIRED.
 
-| **Key name**     | **Description**                                                                                                                                                                                                                                                                                                          |
-| ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| SpatialReference | RECOMMENDED if the derivative is aligned to a standard template listed in [Standard template identifiers][templates]. REQUIRED otherwise. For images with a single reference, the value MUST be a single string. For images with multiple references, such as surface and volume references, a JSON object MUST be used. |
+| **Key name**     | **Requirement level**                                                                                                                     | **Data type**            | **Description**                                                                                                                                                                                    |                                                                                                                       |
```

Comment:
[Rendering problem](https://bids-specification--605.org.readthedocs.build/en/605/05-derivatives/02-common-data-types.html#spatial-references) with this one. I think you have too many columns in this header line.

```suggestion
| **Key name**     | **Requirement level**                                                                                                                     | **Data type**            | **Description**                                                                                                                                                                                    |
```

Reply:
thanks for the catch, updated in code